compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -321,11 +334,16 @@ void addGPUMatmulSimtPassPipeline(OpPassManager &funcPassManager,
     funcPassManager.addPass(createGPUReduceBankConflictsPass());
   }
 
-  if (options.enableReorderWorkgroups) {
+  if (options.reorderStrategy) {
+    funcPassManager.addPass(createReorderWorkgroups(
+        options.reorderStrategy.value(), clReorderWorkgroupsLogSwizzleTile,
```

Comment:
We could have a helper function `getWokrgroupReoderingStrategy` that takes `options` and returns either the workgroup reordering strategy if set, or the global flag. This will reduce branching in this code and the code below.